### ISSUE TITLE
Newbie question

### ISSUE THREAD

**Issue by btcseminar** [CarlRaymond/jquery.cardswipe#29]
I'm posting this as a newbie and hope I'm not out of line, here goes.
I have found a jquery slot machine web app https://github.com/josex2r/jQuery-SlotMachine
When a user taps a button the reels spin like a slot machine.
I run a coffee shop, my aim is to issue "Coffee Membership" cards.
I will display the jquery slot machine in a Samsung  tablet on my counter.
The tablet will have a card reader attached.
When a member pays they will be invited to swipe their card which will fire an event to spin the wheels to win a free coffee.
The cards will not be payment cards.
So my question is, would this be doable?
If the answer is yes, I will pursue finding a developer to build the application for me.
Thanks in advance for any guidance.
@CarlRaymond

**Comment by twf-nikhila** [comment]
@btcseminar, yes this can be done using this plugins. Excellent use case. If you're looking for someone do to it then you can contact me.
